test/sharness/t0170-dht.sh

```diff
@@ -7,10 +7,11 @@ test_description="Test dht command"
 # start iptb + wait for peering
 NUM_NODES=5
 test_expect_success 'init iptb' '
-  iptb init -n $NUM_NODES --bootstrap=none --port=0 &&
-  startup_cluster $NUM_NODES
+  iptb init -n $NUM_NODES --bootstrap=none --port=0
 '
 
+startup_cluster $NUM_NODES
```

Comment:
It's explained in the commit message `startup_cluster()` already contains some `test_expect_success`, so it will already print "ok ..." or "nok ..." for things it checks, see: https://github.com/ipfs/go-ipfs/blob/master/test/sharness/lib/iptb-lib.sh#L20

For example the output is now:

```
> ./t0170-dht.sh 
docker is /usr/bin/docker
ok 1 - init iptb
ok 2 - start up nodes
ok 3 - connect nodes to eachother
ok 4 - node 0 is connected
ok 5 - node 1 is connected
ok 6 - node 2 is connected
ok 7 - node 3 is connected                           
ok 8 - node 4 is connected
ok 9 - peer ids
ok 10 - findpeer
ok 11 - put
ok 12 - findprovs
ok 13 - get
ok 14 - query
ok 15 - stop iptb
# passed all 15 test(s)
1..15
 ```

and tests 2 to 8 are checked inside startup_cluster.